browser/ui/views/brave_tooltips/brave_tooltip_popup_handler.cc

```diff
@@ -31,9 +31,10 @@ void BraveTooltipPopupHandler::Show(Profile* profile,
   DCHECK(tooltip);
 
   const std::string tooltip_id = tooltip->id();
-  DCHECK(!tooltip_popups_[tooltip_id]);
-  tooltip_popups_[tooltip_id] =
-      new brave_tooltips::BraveTooltipPopup(profile, std::move(tooltip));
+  if (!tooltip_popups_[tooltip_id]) {
```

Comment:
I kept running into this DCHECK while testing. I think there's something odd in the staging environment that keeps trying to show the tooltip over and over. In any case, it seems like it's not an invariant that we can hold.